pico_w/wifi/CMakeLists.txt

```diff
@@ -20,9 +20,10 @@ else()
     add_subdirectory_exclude_platforms(http_client)
     add_subdirectory_exclude_platforms(mqtt)
 
-    if (NOT PICO_MBEDTLS_PATH)
-        message("Skipping tls examples as PICO_MBEDTLS_PATH is not defined")
+    if (NOT TARGET pico_mbedtls)
+        message("Skipping tls_client and ota_update examples as Mbed TLS is not available")
     else()
         add_subdirectory_exclude_platforms(tls_client)
+        add_subdirectory_exclude_platforms(ota_update rp2040)
```

Comment:
note ota_update doesn't seem to declare pico_mbedtls as a dependency; if it needs it it probably should

Reply:
That check is being used similar to the encrypted examples, to check that picotool has been compiled with mbedtls - it’s not directly used by the example